app/Models/User.php

```diff
@@ -185,14 +185,28 @@ public function participantInvitations(): Collection
         ])->get();
     }
 
-    public function introduction(): string
+    public function introduction(): array
     {
         return match ($this->context) {
-            'individual' => __('Video for individuals.'),
-            'organization' => __('Video for community organizations.'),
-            'regulated-organization' => __('Video for regulated organizations.'),
-            'training-participant' => __('Video for training participants.'),
-            default => '',
+            'individual' => [
+                'en' => 'https://vimeo.com/850314990/05587fe4df?share=copy',
+                'fr' => 'https://vimeo.com/850319076/4d973fc4ee?share=copy',
+                'asl' => 'https://vimeo.com/850314990/05587fe4df?share=copy',
+                'lsq' => 'https://vimeo.com/850322469/cd5616567a?share=copy',
+            ],
+            'organization' => [
+                'en' => 'https://vimeo.com/850315035/87b6129a8b?share=copy',
+                'fr' => 'https://vimeo.com/850319102/c118d69046?share=copy',
+                'asl' => 'https://vimeo.com/850315035/87b6129a8b?share=copy',
+                'lsq' => 'https://vimeo.com/850322511/2aad27699a?share=copy',
+            ],
+            'regulated-organization' => [
+                'en' => 'https://vimeo.com/850315068/bc26c699cb?share=copy',
```

Comment:
The URL for the video is incorrect as it's a duplicate of the ASL video. The correct URL is `https://vimeo.com/850308924/cab1e34418?share=copy`